packages/amplify-category-api/package.json

```diff
@@ -12,10 +12,17 @@
   "main": "lib/index.js",
   "scripts": {
     "build": "tsc",
+    "watch": "tsc -w",
     "clean": "rimraf lib tsconfig.tsbuildinfo",
     "test": "jest"
   },
   "dependencies": {
+    "@aws-cdk/aws-apigateway": "~1.67.0",
```

Comment:
I have bumped up the CDK version to latest in my last commit. May be we should do the same here